.circleci/config.yml

```diff
@@ -57,9 +47,6 @@ workflows:
       - hmpps/build_docker:
           context: [hmpps-common-vars]
           name: build_docker
-          jira_update: true
-          pipeline_id: <<pipeline.id>>
-          pipeline_number: <<pipeline.number>>
```

Comment:
seems unnecessary to notify JIRA when a docker image is built - the deploy step notifications are more useful